### PR TITLE
[PHPSTAN] Fix: PHPDoc tag var has invalid value

### PR DESCRIPTION
**Pull request by blankse**
This PR fix all `PHPDoc tag @var has invalid value` phpstan level 2 errors.

Before:
`[ERROR] Found 4157 errors`

After:
`[ERROR] Found 3867 errors`

### CODE REVIEW

File: lib/Translation/ExportDataExtractorService/DataExtractor/DataObjectDataExtractor.php [hunk 1]
```diff
@@ -223,28 +223,28 @@ protected function addBlocks(
...
             if ($fd instanceof DataObject\ClassDefinition\Data\Block) {
 
-                /** @var $blockLocalizedFieldDefinition DataObject\ClassDefinition\Data\Localizedfields */
+                /** @var DataObject\ClassDefinition\Data\Localizedfields $blockLocalizedFieldDefinition */
                 $blockLocalizedFieldDefinition = $fd->getFielddefinition('localizedfields');
                 if ($blockLocalizedFieldDefinition) {
                     $blockLocalizedFieldsDefinitions = $blockLocalizedFieldDefinition->getFieldDefinitions();
 
-                    /** @var $blockItems array */
+                    /** @var array $blockItems */
```
Comment (dvesh3) [inline lib/Translation/ExportDataExtractorService/DataExtractor/DataObjectDataExtractor.php]: $blocks

File: lib/Translation/ExportDataExtractorService/DataExtractor/DataObjectDataExtractor.php [hunk 2]
```diff
@@ -223,28 +223,28 @@ protected function addBlocks(
...
                     $blocks = $object->{'get' . ucfirst($fd->getName())}();
 
                     if ($blocks) {
-                        /** @var $blockItem DataObject\Data\BlockElement */
+                        /** @var DataObject\Data\BlockElement $blockItem */
```
Comment (dvesh3) [inline lib/Translation/ExportDataExtractorService/DataExtractor/DataObjectDataExtractor.php]: $blockItems

File: lib/Translation/ImporterService/Importer/DataObjectImporter.php [hunk 3]
```diff
@@ -60,12 +60,12 @@ protected function importAttribute(Element\ElementInterface $element, string $ta
 
         if ($attribute->getType() === Attribute::TYPE_BLOCK_IN_LOCALIZED_FIELD) {
             list($blockName, $blockIndex, $fieldname, $sourceLanguage) = explode(DataObjectDataExtractor::BLOCK_DELIMITER, $attribute->getName());
-            /** @var $blockData array */
+            /** @var array $blockData */
```
Comment (dvesh3) [inline lib/Translation/ImporterService/Importer/DataObjectImporter.php]: $originalBlockData

File: models/DataObject/ClassDefinition/Data.php [hunk 4]
```diff
@@ -1245,11 +1247,11 @@ protected function getDataFromObjectParam($object, $params = [])
                     $container = $object->$containerGetter();
                     if ($container) {
                         $brickGetter = 'get' . ucfirst($context['containerKey']);
-                        /** @var $brickData DataObject\Objectbrick\Data\AbstractData */
+                        /** @var DataObject\Objectbrick\Data\AbstractData $brickData */
                         $brickData = $container->$brickGetter();
 
                         if ($brickData) {
-                            /** @var $localizedFields DataObject\Localizedfield */
+                            /** @var DataObject\Localizedfield $localizedFields */
```
Comment (dvesh3) [inline models/DataObject/ClassDefinition/Data.php]: $data

File: models/DataObject/ClassDefinition/Data/Classificationstore.php [hunk 5]
```diff
@@ -987,7 +987,7 @@ public function checkValidity($data, $omitMandatoryCheck = false)
                         continue;
                     }
 
-                    /** @var $keyGroupRelation DataObject\Classificationstore\KeyGroupRelation */
+                    /** @var DataObject\Classificationstore\KeyGroupRelation $keyGroupRelation */
```
Comment (dvesh3) [inline models/DataObject/ClassDefinition/Data/Classificationstore.php]: move down please

File: models/DataObject/Objectbrick/Definition.php [hunk 6]
```diff
@@ -263,7 +263,7 @@ public function save($saveDefinitionFile = true)
             foreach ($this->getFieldDefinitions() as $key => $def) {
 
                 /**
-                 * @var $def DataObject\ClassDefinition\Data
+                 * @var DataObject\ClassDefinition\Data $def
```
Comment (dvesh3) [inline models/DataObject/Objectbrick/Definition.php]: single line and move down please

File: bundles/AdminBundle/Resources/views/SearchAdmin/Search/Quicksearch/object.html.php [hunk 7]
```diff
@@ -74,13 +74,13 @@
...
                 }
-                /** @var $groupDefinition DataObject\Classificationstore\GroupConfig */
+                /** @var DataObject\Classificationstore\GroupConfig $groupDefinition */
                 $groupDefinition = Pimcore\Model\DataObject\Classificationstore\GroupConfig::getById($activeGroupId);
                 if (!$groupDefinition) {
                     continue;
                 }
 
-                /** @var $keyGroupRelation DataObject\Classificationstore\KeyGroupRelation */
+                /** @var DataObject\Classificationstore\KeyGroupRelation $keyGroupRelation */
```
Comment (dvesh3) [inline bundles/AdminBundle/Resources/views/SearchAdmin/Search/Quicksearch/object.html.php]: move down please

File: bundles/EcommerceFrameworkBundle/PaymentManager/Payment/OGone.php [hunk 8]
```diff
@@ -180,7 +180,7 @@ public function initPayment(PriceInterface $price, array $config)
             'attr' => ['id' => 'payment_ogone_form']
         ]);
 
-        /** @var $paymentInfo \OnlineShop\Framework\Model\AbstractPaymentInformation $paymentInfo * */
+        /** @var \OnlineShop\Framework\Model\AbstractPaymentInformation $paymentInfo $paymentInfo * */
```
Comment (dvesh3) [inline bundles/EcommerceFrameworkBundle/PaymentManager/Payment/OGone.php]: duplicates

Comment (blankse) [reply bundles/EcommerceFrameworkBundle/PaymentManager/Payment/OGone.php]: I removed one

File: bundles/AdminBundle/Resources/views/SearchAdmin/Search/Quicksearch/object.html.php (hunk 7 above)
Comment (blankse) [reply bundles/AdminBundle/Resources/views/SearchAdmin/Search/Quicksearch/object.html.php]: I changed it

File: models/DataObject/Objectbrick/Definition.php (hunk 6 above)
Comment (blankse) [reply models/DataObject/Objectbrick/Definition.php]: I removed it. I set a return Type to getFieldDefinitions()

File: models/DataObject/ClassDefinition/Data/Classificationstore.php (hunk 5 above)
Comment (blankse) [reply models/DataObject/ClassDefinition/Data/Classificationstore.php]: I moved it

File: models/DataObject/ClassDefinition/Data.php (hunk 4 above)
Comment (blankse) [reply models/DataObject/ClassDefinition/Data.php]: I renamed it

File: lib/Translation/ImporterService/Importer/DataObjectImporter.php (hunk 3 above)
Comment (blankse) [reply lib/Translation/ImporterService/Importer/DataObjectImporter.php]: I renamed it

File: lib/Translation/ExportDataExtractorService/DataExtractor/DataObjectDataExtractor.php (hunk 2 above)
Comment (blankse) [reply lib/Translation/ExportDataExtractorService/DataExtractor/DataObjectDataExtractor.php]: I think this was correct, but on the wrong line. I changed it

File: lib/Translation/ExportDataExtractorService/DataExtractor/DataObjectDataExtractor.php (hunk 1 above)
Comment (blankse) [reply lib/Translation/ExportDataExtractorService/DataExtractor/DataObjectDataExtractor.php]: I changed it

File: bundles/EcommerceFrameworkBundle/OfferTool/DefaultService.php [hunk 9]
```diff
@@ -130,9 +130,7 @@ protected function getNewOfferObject($tempOfferNumber)
     {
         $offer = new $this->offerClass();
 
-        /**
-         * @var $offer AbstractOffer
-         */
+        /** @var AbstractOffer $offer */
```
[the remark this answers is not in the document]
Comment (blankse) [reply bundles/EcommerceFrameworkBundle/OfferTool/DefaultService.php]: I moved it up

File: bundles/AdminBundle/Resources/views/Admin/DataObject/DataObject/previewVersion.html.php [hunk 10]
```diff
@@ -140,13 +140,13 @@
...
                 }
-                /** @var $groupDefinition DataObject\Classificationstore\GroupConfig */
+                /** @var DataObject\Classificationstore\GroupConfig $groupDefinition */
                 $groupDefinition = Pimcore\Model\DataObject\Classificationstore\GroupConfig::getById($activeGroupId);
                 if (!$groupDefinition) {
                     continue;
                 }
 
-                /** @var $keyGroupRelation DataObject\Classificationstore\KeyGroupRelation */
+                /** @var DataObject\Classificationstore\KeyGroupRelation $keyGroupRelation */
```
[the remark this answers is not in the document]
Comment (blankse) [reply bundles/AdminBundle/Resources/views/Admin/DataObject/DataObject/previewVersion.html.php]: I moved it down

File: bundles/AdminBundle/Controller/Admin/DataObject/ClassController.php [hunk 11]
```diff
@@ -1642,7 +1642,7 @@ public function doBulkExportAction(Request $request)
                 $result['objectbrick'][] = $objectBrickJson;
             } elseif ($item['type'] == 'customlayout') {
                 $customLayout = DataObject\ClassDefinition\CustomLayout::getById($item['name']);
-                /** @var $customLayout DataObject\ClassDefinition\CustomLayout */
+                /** @var DataObject\ClassDefinition\CustomLayout $customLayout */
```
[the remark this answers is not in the document]
Comment (blankse) [reply bundles/AdminBundle/Controller/Admin/DataObject/ClassController.php]: I moved it up

File: bundles/AdminBundle/EventListener/AdminExceptionListener.php [hunk 12]
```diff
@@ -141,7 +141,7 @@ protected function recursiveAddValidationExceptionSubItems($items, &$message, &$
         if (!$items) {
             return;
         }
-        /** @var $items ValidationException[] */
+        /** @var ValidationException[] $items */
```
[the remark this answers is not in the document]
Comment (blankse) [reply bundles/AdminBundle/EventListener/AdminExceptionListener.php]: I add it to the method phpdocs

File: bundles/AdminBundle/Controller/Admin/Asset/AssetHelperController.php [hunk 13]
```diff
@@ -883,7 +881,7 @@ public function getMetadataForColumnConfigAction(Request $request)
             $uniqueKey = $item->getName().'_'.$item->getTargetSubtype();
             if (!in_array($uniqueKey, $tmp) && !in_array($item->getName(), $defaultMetadataNames)) {
                 $tmp[] = $uniqueKey;
-                /** @var $item Metadata\Predefined */
+                /** @var Metadata\Predefined $item */
```
[the remark this answers is not in the document]
Comment (blankse) [reply bundles/AdminBundle/Controller/Admin/Asset/AssetHelperController.php]: I moved it up

File: bundles/AdminBundle/Controller/Admin/DataObject/ClassController.php [hunk 14]
```diff
@@ -1298,7 +1298,7 @@ public function objectbrickListAction(Request $request)
             $className = $classDefinition->getName();
 
             foreach ($list as $type) {
-                /** @var $type DataObject\Objectbrick\Definition */
+                /** @var DataObject\Objectbrick\Definition $type */
```
[the remark this answers is not in the document]
Comment (blankse) [reply bundles/AdminBundle/Controller/Admin/DataObject/ClassController.php]: I removed it

File: bundles/AdminBundle/Controller/Admin/DataObject/ClassController.php [hunk 15]
```diff
@@ -1181,11 +1181,11 @@ public function objectbrickTreeAction(Request $request)
             $className = $classDefinition->getName();
         }
 
-        /** @var $item DataObject\Objectbrick\Definition */
+        /** @var DataObject\Objectbrick\Definition $item */
         foreach ($list as $item) {
             if ($request->query->has('class_id') && $request->query->has('field_name')) {
                 $keep = false;
-                /** @var $type DataObject\Objectbrick\Definition */
+                /** @var DataObject\Objectbrick\Definition $type */
```
[the remark this answers is not in the document]
Comment (blankse) [reply bundles/AdminBundle/Controller/Admin/DataObject/ClassController.php]: I removed it

File: bundles/AdminBundle/Resources/views/Admin/DataObject/DataObject/diffVersions.html.php [hunk 16]
```diff
@@ -136,13 +136,13 @@
...
                 }
-                /** @var $groupDefinition DataObject\Classificationstore\GroupConfig */
+                /** @var DataObject\Classificationstore\GroupConfig $groupDefinition */
                 $groupDefinition = Pimcore\Model\DataObject\Classificationstore\GroupConfig::getById($activeGroupId);
                 if (!$groupDefinition) {
                     continue;
                 }
 
-                /** @var $keyGroupRelation DataObject\Classificationstore\KeyGroupRelation */
+                /** @var DataObject\Classificationstore\KeyGroupRelation $keyGroupRelation */
```
[the remark this answers is not in the document]
Comment (blankse) [reply bundles/AdminBundle/Resources/views/Admin/DataObject/DataObject/diffVersions.html.php]: I moved it down

File: bundles/AdminBundle/Resources/views/Admin/DataObject/DataObject/diffVersions.html.php [hunk 17]
```diff
@@ -95,7 +95,7 @@
             <?php } ?>
         <?php } else if($definition instanceof DataObject\ClassDefinition\Data\Classificationstore){
 
-            /** @var $storedata DataObject\Classificationstore */
+            /** @var DataObject\Classificationstore $storedata */
```
[the remark this answers is not in the document]
Comment (blankse) [reply bundles/AdminBundle/Resources/views/Admin/DataObject/DataObject/diffVersions.html.php]: I renamed it

File: models/DataObject/ClassDefinition/Data/Block.php [hunk 18]
```diff
@@ -173,10 +170,8 @@ public function getDataFromResource($data, $object = null, $params = [])
 
             foreach ($unserializedData as $blockElements) {
                 $items = [];
-                /** @var $blockElement DataObject\Data\BlockElement */
+                /** @var DataObject\Data\BlockElement $blockElement */
```
Comment (dvesh3) [inline models/DataObject/ClassDefinition/Data/Block.php]: $blockElementRaw

File: models/DataObject/ClassDefinition/Data/Block.php [hunk 19]
```diff
@@ -508,12 +496,8 @@ public function getFromWebserviceImport($value, $relatedObject = null, $params =
             foreach ($value as $blockElementsData) {
                 $resultElement = [];
 
-                /**
-                 * @var  $blockElement DataObject\Data\BlockElement
-                 */
+                /** @var DataObject\Data\BlockElement $blockElement */
```
Comment (dvesh3) [inline models/DataObject/ClassDefinition/Data/Block.php]: $blockElementDataRaw

File: models/DataObject/Fieldcollection/Data/AbstractData.php [hunk 20]
```diff
@@ -103,7 +103,7 @@ public function getType()
     }
 
     /**
-     * @return mixed
+     * @return Model\DataObject\Fieldcollection\Definition
```
Comment (dvesh3) [inline models/DataObject/Fieldcollection/Data/AbstractData.php]: Definition|null

Comment (blankse) [reply models/DataObject/Fieldcollection/Data/AbstractData.php]: Do you think we cannot be sure that the definition exists? Than we have to check it on any usage with an if. Now there is no if at any.

File: models/DataObject/ClassDefinition/Data/Block.php (hunk 19 above)
Comment (blankse) [reply models/DataObject/ClassDefinition/Data/Block.php]: $blockElementDataRaw should be a array. $blockElement is a few lines below, we can remove this.

File: models/DataObject/ClassDefinition/Data/Block.php (hunk 18 above)
Comment (blankse) [reply models/DataObject/ClassDefinition/Data/Block.php]: $blockElementDataRaw should be a array. $blockElement is a few lines below, we can remove this.

File: models/DataObject/Fieldcollection/Data/AbstractData.php (hunk 20 above)
Comment (dvesh3) [reply models/DataObject/Fieldcollection/Data/AbstractData.php]: good point!...I was just looking at return values from `getByKey` method as it still returns null. please revert the null change and sorry for rework.

Comment (blankse) [reply models/DataObject/Fieldcollection/Data/AbstractData.php]: Okay, I reverted it.

**Pull request merged**